wallet/storage/src/internal/test.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::*;
+
+type TestStore = super::Store<storage_sqlite::Sqlite>;
+
+#[test]
+fn test_storage_get_default_version_in_tx() {
+    utils::concurrency::model(|| {
```

Comment:
I think you need to import the utils crate in Cargo.toml

Reply:
I somehow commit but didn't push the changes